lib/freno/client.rb

```diff
@@ -5,6 +5,7 @@
 
 module Freno
   class Client
+    class DecorationError < StandardError; end
```

Comment:
Should this extend `ArgumentError` instead? WDYT about moving it to the `errors.rb` file? Maybe not?

Reply:
Switched it to `ArgumentError` 👍 It felt a little weird moving it to `errors.rb` because that's nested a level or two down, and not currently included in this file